pinot-common/src/main/java/org/apache/pinot/common/config/TagNameUtils.java

```diff
@@ -45,7 +46,18 @@ public static boolean hasValidServerTagSuffix(String tagName) {
     return false;
   }
 
-  public static TenantRole getTenantRoleFromTag(String tagName) throws InvalidConfigException {
+  public static boolean isServerTag(String tagName)
+      throws InvalidConfigException {
+    return TenantRole.SERVER == getTenantRoleFromTag(tagName);
+  }
+
+  public static boolean isBrokerTags(String tagName)
+      throws InvalidConfigException {
+    return TenantRole.BROKER == getTenantRoleFromTag(tagName);
+  }
+
+  // Make this method private to avoid exposing null out of this class.
+  private static TenantRole getTenantRoleFromTag(String tagName) throws InvalidConfigException {
```

Comment:
I think one of the reasons of keeping that is `tagName` can be misconfigured. This method `isServerTag` is to check whether it's a wrong tag name. Otherwise it will just return `false` and provides no extra information, which makes it harder to figure out.

Reply:
Discussed with @Jackie-Jiang. The exception should have been caught when table config was added/updated. So there's no need to keep the validation check of tag name here.